packages/diffast-cli/diffast-cli.0.3.6/opam

```diff
@@ -0,0 +1,52 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Diff/AST command line interface"
+description:
+  "Diff/AST command line interface. Diff/AST is a fine-grained source code differencing tool based on an algorithm for computing tree edit distance (TED) between two ordered labeled trees."
+maintainer: [
+  "Codinuum <codinuum@me.com>"
+  "Masatomo Hashimoto <m.hashimoto@stair.center>"
+]
+authors: [
+  "Codinuum <codinuum@me.com>"
+  "Masatomo Hashimoto <m.hashimoto@stair.center>"
+]
+license: "Apache-2.0"
+homepage: "https://github.com/codinuum/diffast"
+doc: "https://github.com/codinuum/diffast/README.md"
+bug-reports: "https://github.com/codinuum/diffast/issues"
+depends: [
+  "ocaml" {>= "4.14"}
+  "dune" {>= "3.18"}
+  "diffast-misc" {= version}
+  "diffast-core" {= version}
+  "diffast-api" {= version}
+  "diffast-etc" {= version}
+  "vlt" {>= "0.2.4"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "--promote-install-files=false"
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+  ["dune" "install" "-p" name "--create-install-files" name]
+]
+dev-repo: "git+https://github.com/codinuum/diffast.git"
+x-maintenance-intent: ["(latest)"]
+url {
+  src: "https://github.com/codinuum/diffast/archive/v0.3.6.tar.gz"
+  checksum: [
+    "sha256=be973fe1316a3d72672c03680305ca8a4ad947ed07a8c3eeea8380faec9e52c0"
+    "md5=edb85aaf77cd22aaa8f6ce3889bf241d"
+  ]
+}
```

Comment:
```suggestion
}
available: os != "win32"  # temporarily silence file clash errors on Cygwin/MSys2 MinGW
```
OK, thanks both!

Now that there's a reproducer and a way forward for both `dune` and `diffast` I suggest marking the latter unavailable for this release (... and perhaps the previous?) until the `install` stanza has been reworked in the next release.